src/rpc/governance.cpp

```diff
@@ -133,12 +133,15 @@ void gobject_prepare_help()
                 "3. time          (numeric, required) time this object was created\n"
                 "4. data-hex      (string, required)  data in hex string form\n"
                 "5. use-IS        (boolean, optional, default=false) InstantSend lock the collateral, only requiring one chain confirmation\n"
+                "6. outputHash    (string, optional) the single output to submit the fee from\n"
+                "7. outputIndex   (numeric, optional) The output index.\n"
                 );
+    );
```

Comment:
```suggestion
```

Reply:
Weird, I swear I removed that in the merge 🤷‍♂️